### ISSUE TITLE
[Bug]: Authentication Partially Completes, Music and Playlists/Playlist Cache fail to Load

### ISSUE THREAD

**Issue by AyoungDukie** [ciderapp/Cider-2#283]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

BUILD_DATE: '2023-05-17T05:25:48.852Z', APP_VERSION: `2.0.3`, GIT_COMMIT: `91d26a65a3dc310c7e68fef9d3d03144ce9de937`

### Operating System Version

Windows 10 version 21H2, build 19044.3086

### Describe the Bug

This appears related to #170 and discussions in #279; I have been able to work around the second issue by pre-logging in to the web app for Apple Music in Edge; this seems to allow the token authentication page to complete loading.

Going through the process seems to only partially load, however with an error occurring on the last screen:
![image](https://github.com/ciderapp/Cider-2/assets/5449443/423c9afa-e43a-4b11-ab09-ef98790db873)
and the following console outputs:
![image](https://github.com/ciderapp/Cider-2/assets/5449443/d1926cea-f485-44b3-8b05-02360263eeb1)

Clicking continue will bring up the app, which now acts as though it is authenticated. However, once in, songs and playlists fail to load. This seems to be due to the errors/warnings displaying in the dev console:
![image](https://github.com/ciderapp/Cider-2/assets/5449443/d137d0b9-743a-4f73-8074-162f6318d9d3)

The last two errors (404 and Uncaught (in promise)... ) then repeat periodically while the playlists fail to load.

Attempting to play a song yields the following sequence:
![image](https://github.com/ciderapp/Cider-2/assets/5449443/1a3c0fdd-389a-4970-8756-318e2eb1bd75)

Where again the final two 404 errors reoccur a few times.

### Steps to Reproduce

Log out of Apple Music  and try to log in, or attempt a first log-in.

### Build timestamp

1688586556676

**Comment by marclallen** [comment]
I'm also seeing this problem.  I upgraded my system a couple of weeks ago.  I also got the network connection error, but it eventually gave me playlists and such.  I was using it fine.  Today, after not using it for a week or more, it no longer loads any music and I still get the strange login issue.

**Comment by AyoungDukie** [comment]
As noted in #284, this should be addressed when 2.1 releases. Will close here for now, and can reopen should it continue to be an issue.

**Comment by maxwiseman** [comment]
Updated to `2.1.0` via Microsoft Store and I'm still having this issue. I see that I can now use a token, but I have no easy way to actually get a token (as far as I know). Still seeing the same errors. I can log in and my music will show, but I cannot actually play it. 

I have tried resetting the app's data via Windows Settings, I reset all of my browsing history and cookies in Edge, I tried logging into my Apple account via a Passkey, I tried logging into the music.apple.com website, I tried copying the authentication URL into Edge (results in `p55-buy.itunes.apple.com redirected you too many times`), I tried reinstalling the app, and I tried creating an app password and using that.
